tXenqueue/check_posted_tx_payload.py

```diff
@@ -13,7 +13,10 @@
                         'page_margins', 'page_size', 'toc_levels'
 
 # NOTE: The following are currently only used to log warnings -- they are not strictly enforced here
-KNOWN_RESOURCE_TYPES = 'bible', 'obs', 'ta', 'tn', 'tq', 'tw', 'ult', 'ust', 'ugnt', 'uhb', 'ulb'
```

Comment:
@jag3773 Not sure quite what you mean? Perhaps I still haven't done exactly what you expected? (I went ahead with the merge coz I was pretty sure that my changes were working -- doesn't necessarily mean that it's working how you wanted it.) The updated Door43 side passes the subject as the resource_type if it matches, otherwise still does it as before. This then gets passed thru to the tX side which chooses the converter and linter. (No new converters/linters have been added yet.)